ci-operator/config/openshift/projected-resource-csi-driver/openshift-projected-resource-csi-driver-master.yaml

```diff
@@ -0,0 +1,30 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+resources:
+  '*':
+    requests:
+      cpu: "3"
+      memory: 1Gi
+tests:
+- artifact_dir: /tmp/artifacts
+  as: verify
+  commands: |
+    mkdir -p /tmp/artifacts
+    make verify > /tmp/artifacts/verify.log
+  container:
+    from: bin
+- artifact_dir: /tmp/artifacts
+  as: unit
+  commands: |
+    mkdir -p /tmp/artifacts/coverage
+    make test OUTPUT_COVERAGE=/tmp/artifacts/coverage > /tmp/artifacts/test-unit.log
```

Comment:
perhaps we should follow this pattern instead:  perhaps we should follow this pattern instead:  https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-openshift-controller-manager-operator/openshift-cluster-openshift-controller-manager-operator-master.yaml#L44-L45

I think the output showing up the in logs is sufficient capture  vs. storing them in a file, no?

Reply:
updated in current push